frontend/taipy-gui/src/context/taipyReducers.ts

```diff
@@ -871,6 +877,13 @@ export const createDeleteNotificationAction = (snackbarId: string): TaipyDeleteN
     }
 }
 
+export const createSendAction = (name: string): TaipySendAction => {
+    return {
+        type: Types.Action,
+        name,
+    };
+};
+
```

Comment:
use `createSendActionNameAction`

Reply:
and add optionals `reason` and `on close` to [NotificationMessage](https://github.com/Avaiga/taipy/blob/0fad04a8e458e3be45d7892d369fb069448dfbec/frontend/taipy-gui/src/context/taipyReducers.ts#L91)